extension/imos--unpackaged--1.0.sql

```diff
@@ -4,24 +4,24 @@
 \echo Use "CREATE EXTENSION imos" to load this file. \quit
 
 -- Harvester
-ALTER EXTENSION "imos" ADD function getendtoken();
-ALTER EXTENSION "imos" ADD function format_name();
-ALTER EXTENSION "imos" ADD function add_point_shortest();
-ALTER EXTENSION "imos" ADD function make_trajectory();
-ALTER EXTENSION "imos" ADD function make_shortest_line();
+ALTER EXTENSION "imos" ADD function public.getendtoken();
```

Comment:
What are these changes doing? I thought unpackaged extensions were a legacy thing we didn't have to deal with any more?

Reply:
yes, these changes aren't necessary, but I was letting that one slide.  the unpackaged--1.0.sql file is for adding the unmanaged functions to the extension in the first place.  that was possibly used a long time ago to add functions probably managed by an sql script at the time to the extension to get us to version 1.0